### PR TITLE
added status to statuses.py

### PR DESCRIPTION
**Pull request by Nikhil-Kudupudi**

### CODE REVIEW

File: statuses.py [hunk 1]
```diff
@@ -65,4 +65,9 @@
    "A hug in time saves nine 🙂.",
    "Hey you! I'm talking to you, yes you! I'm sending you a hug 🤗.",
    "Roses are red,\nViolets are blue,\nYou're so great,\nI just want to hug you!",
+   "A hug makes you feel good all day.",
+   "I love hugging. I wish I was an octopus, So I could hug ten people at a time.",
```
Comment (achasveachas) [inline statuses.py:69]: Minor grammar fix: the comma after "octopus" is unnecessary and the "S" of "so" should be lower-case.

**Review by achasveachas — CHANGES_REQUESTED** [review]
These are great! 🤗 

One minor grammar fix and then I'll merge it.

File: statuses.py (hunk 1 above)
Comment (Nikhil-Kudupudi) [reply statuses.py]: Made the changes

**Review by achasveachas — APPROVED** [review]
Awesome, thanks! 🤗 

If you have a Twitter handle please let me know so I can give you a shout-out 💜

**Pull request merged**